models/ExperimentModel.js

```diff
@@ -1,20 +1,24 @@
 const { prisma } = require('../db');
 const { API_BASE_URL } = require('../env');
 
-// const findMany = (locationId, timestamp) =>
-//   prisma.experiment.findMany({
-//     where: { locationId: parseInt(locationId, 10), timestamp },
-//   });
-
-const findExperimentByTimestamp = async (locationId, timestamp) => {
-  const [response] = await prisma.experiment.findMany({
+const findExperimentByTimestamp = async (locationId, timestamp) =>
+  prisma.experiment.findFirst({
     where: {
       locationId: parseInt(locationId, 10),
       timestamp,
     },
   });
-  return response;
-};
+
+const findLatestExperiment = async (locationId) =>
```

Comment:
You declared an async function but you did not use await in it.
you should add await after "=>" or delet the async statement.